### ISSUE TITLE
Installation from master not detected by MDAnalysis

### ISSUE THREAD

**Issue by RMeli** [MDAnalysis/GridDataFormats#66]
The installation of GridDataFormats from the `master` branch (using `pip install .` in `GridDataFormats-master`)
```
Processing /users/USER/software/python/GridDataFormats-master
Requirement already satisfied: numpy>=1.0.3 in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from GridDataFormats==0+unknown) (1.16.2)
Requirement already satisfied: six in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from GridDataFormats==0+unknown) (1.12.0)
Requirement already satisfied: scipy in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from GridDataFormats==0+unknown) (1.2.1)
Building wheels for collected packages: GridDataFormats
  Building wheel for GridDataFormats (setup.py) ... done
  Stored in directory: /users/USER/.cache/pip/wheels/dc/94/b9/b371f823d3cf2ec1403f27862277becedcc4ecb15441e8e527
Successfully built GridDataFormats
Installing collected packages: GridDataFormats
  Found existing installation: GridDataFormats 0+unknown
    Uninstalling GridDataFormats-0+unknown:
      Successfully uninstalled GridDataFormats-0+unknown
Successfully installed GridDataFormats-0+unknown
```
is note detected when installing MDAnalysis (`pip install -e package` in `mdanalysis-develop`):
```bash
Obtaining file:///users/USER/software/python/mdanalysis-develop/package
Requirement already satisfied: numpy>=1.10.4 in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from MDAnalysis==0.19.3.dev0) (1.16.2)
Requirement already satisfied: biopython>=1.71 in /users/USER/software/python/biopython-master (from MDAnalysis==0.19.3.dev0) (1.74.dev0)
Requirement already satisfied: networkx>=1.0 in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from MDAnalysis==0.19.3.dev0) (2.2)
Collecting GridDataFormats>=0.4.0 (from MDAnalysis==0.19.3.dev0)
  Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7ff1debda5c0>: Failed to establish a new connection: [Errno 101] Network is unreachable')': /simple/griddataformats/
```

However, the tagged version `4.0.1` works as expected when installing MDAnalysis (`pip install -e package` in `mdanalysis-develop`):
```bash
Obtaining file:///users/USER/software/python/mdanalysis-develop/package
Requirement already satisfied: numpy>=1.10.4 in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from MDAnalysis==0.19.3.dev0) (1.16.2)
Requirement already satisfied: biopython>=1.71 in /users/USER/software/python/biopython-master (from MDAnalysis==0.19.3.dev0) (1.74.dev0)
Requirement already satisfied: networkx>=1.0 in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from MDAnalysis==0.19.3.dev0) (2.2)
Requirement already satisfied: GridDataFormats>=0.4.0 in /users/USER/software/python/GridDataFormats-0.4.1 (from MDAnalysis==0.19.3.dev0) (0.4.1)
Requirement already satisfied: six>=1.4.0 in /users/USER/anaconda3/envs/mda/lib/python3.7/site-packages (from MDAnalysis==0.19.3.dev0) (1.12.0)
```

**Comment by orbeckst** [comment]
We use versioneer to autmatically generate the release number. versioneer needs the tags. Did you
```
git pull --tags
```
as well?

Otherwise versioneer can only create a simplified tag that will not pass the requirement of MDAnalysis that GridDataFormats has to be >= 0.4.0 and hence pip will attempt to download from PyPi.

**Comment by RMeli** [comment]
I did not, and pulling the tags solves the problem. Thank you very much for pointing this out!
